content/SCALE/SCALETutorials/Apps/CommunityApps/MinIOApp/_index.md

```diff
@@ -0,0 +1,146 @@
+
+### First Steps
+
+{{< include file="/_includes/MinIODatasetRequirements.md" type="page" >}}
+
+### Configuring MinIO (S3) Community App
+
+{{< include file="/_includes/MinIOInstallAppNameWorkloadConfigSteps.md" type="page" >}}
+
+Next, enter the **MinIO Configuration** settings.
+
+{{< include file="/_includes/MinIOInstallArgAndEnvironVarSteps.md" type="page" >}}
+
+{{< include file="/_includes/MinIOPortsAndLogSearch.md" type="page" >}}
+
+{{< include file="/_includes/MinIOStorageDataVolume.md" type="page" >}}
+
+{{< trueimage src="/images/SCALE/23.10/InstallMinioStorageAddExtraHostPathVol.png" alt="Add Host Path Volume" id="5: Add Host Path Volume" >}}
+
+If you want to create volumes for postgres data and postgres backup, select **Postgres Data Volume** and/or **Postgres Backup Volume** to add the mount and host path fields for each. 
+If not set, TrueNAS uses the defaults for each **postgres-data** and **postgres-backup**.
+
+{{< include file="/_includes/MinIODNSAndResourceLimits.md" type="page" >}}
+
+The **Installed** applications screen displays showing the Minio application in the **Deploying** state. 
+It changes to **Running** when the application is ready to use. 
+
+{{< trueimage src="/images/SCALE/23.10/MinIOAppInstalled.png" alt="Minio App Installed" id="6: Minio App Installed" >}}
+
+Click **Web Portal** to open the MinIO sign-in screen.
+
+{{< trueimage src="/images/SCALE/23.10/MinIOWebPortal.png" alt="MinIO Sign-In Screen" id="7: MinIO Sign-In Screen" >}}
+
+#### Application Name Settings
+Accept the default value or enter a name in **Application Name** field. 
+Accept the default version number in **Version**.
+
+#### Workload Configuration Settings
+The Minio **Workload Configuration** section includes the **Minio update strategy** setting that sets how application updates occur. 
+
+{{< trueimage src="/images/SCALE/23.10/InstallMinioWorkloadConfig.png" alt="Minio Workload Configuration" id="8: Minio Workload Configuration" >}}
+
+Select **Create new pods then kill old ones** to implement a rolling update strategy where the existing container (pod) remains until the update completes, then it is removed. 
+Select **Kill existing pods before creating new ones** to implement a recreate update strategy where you remove the existing container (pod) and then create a new one.
+The recommended option is to keep the default and use the the rolling update strategy.
+
+#### Minio Configuration Settings
+The **Minio Configuration** section provides options to set up a cluster, add arguments, credentials, and environment variables to the deployment. 
+
+{{< include file="/_includes/MinIOEnableDistributedModeInfo.md" type="page" >}}
+
+The app is preconfigured with arguments it needs to deploy a container. Do not enter the **server** and URL argument earlier versions of the app required.
+
+{{< trueimage src="/images/SCALE/23.10/InstallMinioAddConfiguration.png" alt="Minio Configuration Settings" id="9: Minio Configuration Settings" >}}
```

Comment:
Might be a good idea to move this image up to line 90 or so. When I read the first line of the Distributed mode snippet in the local build version I started looking around for an image to see what Enable Distributed Mode looks like

Reply:
This article is not instructing them on creating a distributed mode or cluster configuration so that is why this text is down in the details section. The cluster article has this content in the flow of the procedure.